### DISCUSSION TITLE
Convert God's Grand Temple into Another Deity

### DISCUSSION THREAD

**Discussion by arkimedes1967** [Keriew/augustus#224] [Ideas]
Please exclude this thread.

**Comment by crudelios** [comment]
I'm not sure about everyone else, but the idea for grand temples is that they need to be carefully chosen by the player, so I don't think they should easily be able to change them.

Besides, each temple's construction stage already looks very much like an in-progress build of the final thing, so it would be weird to change to a temple of a different kind

The only thing that could happen on temple deletion is that some of the marble could be returned to the player.

Regarding changing the epitaph: I agree, but at the full 1000 dn cost as well as 2 marble (to restore the walls after removing the previous epitaph and before adding the new one).
